homeassistant/components/flume/sensor.py

```diff
@@ -0,0 +1,93 @@
+"""Sensor for displaying the number of result from Flume."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
```

Comment:
Please remove this blank line so that 3rd party imports are grouped together.

Reply:
I recommend using the `isort` tool to sort and group imports automatically.

Reply:
Thank you for the recommendation!